suite-common/fiat-services/src/coingecko.ts

```diff
@@ -154,9 +154,12 @@ export const getFiatRatesForTimestamps = async (
     const url = `${coinUrl}/${urlEndpoint}${params}`;
 
     // returns pairs of [timestamp, fiatRate]
-    const { prices }: { prices: Array<[number, number]> } = await fetchCoinGecko(url);
+    const response = await fetchCoinGecko(url);
+    if (!response || !response.prices || response.prices.length === 0) {
+        return null;
+    }
 
-    if (!prices || prices.length === 0) return null;
+    const { prices }: { prices: Array<[number, number]> } = response;
```

Comment:
nit: move the type to `fetchCoinGecko()`

Reply:
and there's no need to restructure at this point, just do `findClosestTimestampValue(ts, rates.prices)` :DD

Reply:
bbc05a105ca2900a481c640ebf479a5366a66691 - removed the type since the API is returning results based on parameters and we are using just few (imo not really needed)